i18n/translations/en/common.json

```diff
@@ -54,7 +54,13 @@
     "validGovEmail": "Enter a valid government email address.",
     "notSameAsUserEmail": "This cannot be your email address. Enter the email of the person in charge of your team. Enter the email of the person in charge of your team. If that's you, add a colleague.",
     "min-length-4-characters": "Must be at least 4 characters.",
-    "MB": "MB"
+    "MB": "MB",
+    "error-list": {
+      "default": "Enter an answer for: {{question}}",
+      "select": "Select an option for: {{question}}",
+      "checkbox": "Select one or multiple options for: {{question}}",
+      "check-all": "Check off the boxes for: {{question}}"
```

Comment:
```suggestion
      "default": "Enter an answer for: {{question}}",
      "select": "Select an option for: {{question}}",
      "checkbox": "Select one or multiple options for: {{question}}",
      "check-all": "Check all the boxes for: {{question}}"
```

Reply:
Possible to add "file upload" form element type?

EN: "Choose a file for: {{question}}"
FR: EN: "Choisissez un fichier pour : {{question}}"

Reply:
ya should be able to